app/javascript/elm/src/Main.elm

```diff
@@ -177,8 +177,8 @@ update msg model =
         ShowCopyLinkTooltip ->
             ( model, Ports.showCopyLinkTooltip () )
 
-        ShowExpandableSelectFromPopup ->
-            ( { model | popup = Popup.ExpandableSelectFrom (Sensor.allParametersWithPrioritization model.sensors) }, Cmd.none )
+        ShowExpandableSelectFromPopup items itemType ->
+            ( { model | popup = Popup.ExpandableSelectFrom items itemType, isPopupExtended = False }, Cmd.none )
```

Comment:
Maybe we could have a `popupState : { isExtended : Bool, items : ...}` so that it's clear that both are popup stuff?

Reply:
I'm not sure about that. The popup can have two states (shown or hidden named `SelectFrom`, `None`), that is already in the model under `popup`.  Then only `SelectFrom` is interested in `isExtender` and `items`.
But in general, I think it's a good idea to connect them, just didn't have time to figure it out today.